plugins/cratedb/database_credentials_test.go

```diff
@@ -0,0 +1,35 @@
+package cratedb
+
+import (
+	"testing"
+	
+	"github.com/1Password/shell-plugins/sdk"
```

Comment:
Linter is complaining that this file is not formatted, and more specifically these imports are not in order.